### PR TITLE
Redpitaya patch

### PR DESCRIPTION
**Pull request by seb5g**
This is a patch of the initial PR #1010 modifying the method to fetch data (easier if triggering) and renamed a setting that didn't make sense

### CODE REVIEW

**Comment by seb5g** [comment]
I've build a pymodaq plugin using it which is now fully functional so the acquisition part of the driver should not change much than this

File: pymeasure/instruments/redpitaya/redpitaya_scpi.py [hunk 1]
```diff
@@ -119,14 +119,33 @@ class AnalogInputFastChannel(Channel):
...
     )
 
-    def get_data_from_ascii(self) -> np.ndarray:
-        self.write("ACQ:SOUR{ch}:DATA?")
+    def get_data(self, npts: int = None, format='ASCII') -> np.ndarray:
+        """ Read data from the buffer
+
+        :param npts: number of points to be read
+        :param format: either 'ASCII' or 'BIN', see :meth:acq_format
+        :param center_trigger: if True data are symmetric around the trigger event position
```
Comment (BenediktBurger) [inline pymeasure/instruments/redpitaya/redpitaya_scpi.py:127]: There is no center trigger parameter!

**Review by BenediktBurger — APPROVED** [review]
Thanks for the improvement.

We keep code backward compatible and raise deprecation warnings before removal, if there is a removal at all.

File: pymeasure/instruments/redpitaya/redpitaya_scpi.py (hunk 1 above)
Comment (seb5g) [reply pymeasure/instruments/redpitaya/redpitaya_scpi.py]: done

**Comment by BenediktBurger** [comment]
May I merge the changes?

**Comment by seb5g** [comment]
Yes please

**Pull request merged**